lib/Zabbix/Tiny.pm

```diff
@@ -40,45 +40,73 @@ has 'request'         => ( is => 'ro' );
 has 'json_prepared'   => ( is => 'ro' );
 has 'json_executed'   => ( is => 'ro', default => sub { 0 } );
 has 'redo'            => ( is => 'ro' );
+has 'version'         => ( is => 'rw' );
```

Comment:
Is there a reason `version` is writable?  I would think this should be set at instantiation. It might also benefit from a default value.

Reply:
No reason. Switching to ro. 
Not agree about default value tho. Why would we want to guess the version of Zabbix server via default value if we instead can just get it from API. We also still leave the ability for user to explicitly supply the version of server, in which case we will not attempt to detect it via API call if some one see it as an issue.